src/components/DeliverablesTable/index.tsx

```diff
@@ -40,17 +43,28 @@ const DeliverablesTable = ({
   filterModifiers,
   isAcceleratorRoute,
   organizationId,
+  participantId,
   searchAndSort,
   tableId,
 }: DeliverablesTableProps) => {
   const dispatch = useAppDispatch();
   const { activeLocale } = useLocalization();
   const { availableProjects: projects, getProjectName } = useProjects();
+  const { selectedParticipant } = useParticipants(participantId);
 
   const [deliverables, setDeliverables] = useState<ListDeliverablesElement[]>([]);
   const [deliverablesSearchRequestId, setDeliverablesSearchRequestId] = useState('');
   const deliverablesSearchRequest = useAppSelector(selectDeliverablesSearchRequest(deliverablesSearchRequestId));
 
+  const getFilterProjectName = useCallback(
```

Comment:
Can you throw this into the `useParticipants` hook?

Reply:
You got it 👍

Reply:
@graz just moved that over, it depended on `selectedParticipant` and `getProjectName` so I opted to pass them both in. Not sure if that's what you were thinking?

Reply:
I sent you down the wrong path, since we are essentially combining two hooks, I think it was cleaner the way you originally had it. Sorry for the run around!

Reply:
No worries 😄